src/main/java/fruitshop/FruitShop.java

```diff
@@ -0,0 +1,5 @@
+package fruitshop;
+
+public interface FruitShop {
+    void createNewReport(String fileName, String nameNewReport);
```

Comment:
The method with the name `createReport()` must only create a report without side effects
```suggestion
    String createReport();
```